definitions/features/katello_service.rb

```diff
@@ -0,0 +1,28 @@
+class Features::KatelloService < ForemanMaintain::Feature
+  metadata do
+    label :katello_service
+  end
+
+  def make_stop(spinner, args = '')
+    originally_running_services = running_services
+    spinner.update 'Stopping katello running services..'
+    execute!("katello-service stop #{args}")
```

Comment:
@iNecas , 
Could you please elaborate 2nd point as I am bit confuse here.
As you mentioned firstly I need to rename args as `services` then 
- Instead of passing exclude list of services, I have to pass services which needs to be stop. 
- After that I need to compare `running_services` with `services`, 
   stop only services which are common in both list.

Please correct me.